### PR TITLE
Docs: added keyword MAXIMUM_POWER in eCalc YAML keyword list.

### PR DESCRIPTION
**Pull request by equinor-schen**
## Why is this pull request needed?

This pull request is needed because a new keyword MAXIMUM_POWER was added in eCalc YAML keywords.

### CODE REVIEW

File: docs/docs/about/references/keywords/MAXIMUM_POWER.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
+# MAXIMUM_POWER
+
+[MODELS](/about/references/keywords/MODELS.md) / 
+[MAXIMUM_POWER](/about/references/keywords/MAXIMUM_POWER.md)
+
+## Description
+
+`MAXIMUM_POWER` is an optional constand maximum power (MW) that the compressor train can provide.
```
Comment (olelod) [inline docs/docs/about/references/keywords/MAXIMUM_POWER.md:8]:
```suggestion
`MAXIMUM_POWER` is an optional constant giving the maximum power (MW) that the compressor train can use.
```

File: docs/docs/about/references/keywords/MAXIMUM_POWER.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+## Functionality
+
+ It is an optional setting and supported for compressor train models [SIMPLIFIED_VARIABLE_SPEED_COMPRESSOR_TRAIN](/about/modelling/setup/models/compressor_modelling/compressor_models_types/simplified_variable_speed_compressor_train_model.md), [VARIABLE_SPEED_COMPRESSOR_TRAIN](/about/modelling/setup/models/compressor_modelling/compressor_models_types/variable_speed_compressor_train_model.md), [VARIABLE_SPEED_COMPRESSOR_TRAIN_MULTIPLE_STREAMS_AND_PRESSURES](/about/modelling/setup/models/compressor_modelling/compressor_models_types/variable_speed_compressor_train_model_with_multiple_streams_and_pressures.md).
+
```
Comment (olelod) [inline docs/docs/about/references/keywords/MAXIMUM_POWER.md:12]: Also valid for single speed compressor trains

Comment (equinor-schen) [reply docs/docs/about/references/keywords/MAXIMUM_POWER.md:12]: SINGLE_SPEED_COMPRESSOR_TRAIN was added in MAXIMUM_POWER Functionality.

**Comment by equinor-schen** [comment]
@olelod, could you please take a review if the commits can be approved?

**Review by olelod — APPROVED** [review]

**Pull request merged**